STYLE_GUIDE.md

```diff
@@ -0,0 +1,461 @@
+# Pony Standard Library Style Guide
+
+This is a document describing the preferred practices for writing code for the Pony standard library. Other projects are free to use the same style, but you are free to use any style in other projects. The following conventions are meant to ensure consistency and readability of Pony code. Any of the following rules may be broken in situations where the recommended practice would diminish readability.
+
+
+## Formatting
+
+### Line Length
+
+The maximum line length is 80 columns.
```

Comment:
Has there been any discussion of widening this limit to 100 columns? While I understand not letting lines get too wide, 80 always seems so narrow to me. I find 100 to be a sweet spot.